draft-ietf-tls-tls13.md

```diff
@@ -4802,21 +4802,23 @@ by IANA:
 This section provides a summary of the legal state transitions for the
 client and server handshakes.  State names (in all capitals, e.g.,
 START) have no formal meaning but are provided for ease of
-comprehension.  Messages which are sent only sometimes are indicated
-in [].
+comprehension.  Actions which are taken only in certain circumstances are
+indicated in [].
 
 ## Client
 
 ~~~~
                            START <----+
             Send ClientHello |        | Recv HelloRetryRequest
-         /                   v        |
+   [Rekey out to early keys] |        | Send ClientHello
+         /                   v        | 
         |                  WAIT_SH ---+
```

Comment:
Having this line attach to the "[Rekey out...]" line appears to have some significance, can you move the '/' down one line?  It's a fault in the original diagram as well...